S7.Net/AutoReconnector.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Timers;
+
+namespace S7.Net
+{
+    public class AutoReconnector
```

Comment:
The way this is setup is a bad idea. The PLC class is not threadsafe, this will perform `Open` calls from a different thread, it's broken fundamentally.